test/correctness/float_helpers.h

```diff
@@ -0,0 +1,40 @@
+#ifndef FLOAT_HELPERS_H
+#define FLOAT_HELPERS_H
+#include <cinttypes>
+uint32_t bits_from_float(float v) {
```

Comment:
We have a templated bit-caster in Util.h. So of this can probably all just be ```reinterpret_bits<uint32_t>(float)``` and similar.